src/routes/terms-of-service/CallToAction.svelte

```diff
@@ -0,0 +1,65 @@
+<script>
+  import { onMount, createEventDispatcher } from "svelte";
+  import { fade, fly } from "svelte/transition";
+  import Arrow01 from "../../components/Arrow01.svelte";
+  import Button from "../../components/Button.svelte";
+
+  const dispatch = createEventDispatcher();
+
+  let scrollY = 0;
+
+  let showArrow = true;
+  $: if (scrollY > 20) {
+    showArrow = false;
+  } else {
+    showArrow = true;
+  }
+
+  // let mounted = false;
+  // onMount(() => {
+  //   mounted = true;
+  // });
```

Comment:
Same question re: commented code :) If it serves a purpose (for testing etc) then we should have a comment above it at least (or do it in different way preferably)